packages/cdktf-cli/lib/get/constructs-maker.ts

```diff
@@ -249,6 +252,23 @@ export class ConstructsMaker {
             }
           }
 
+          if (this.isGoTarget) {
+            // const targetType = target.isProvider ? 'provider' : 'module';
+            
+            // could also contain e.g. "aws" as abbreviation for "hashicorp/aws" for official providers
+            const parts = target.fqn.split('/'); // TODO: check what could be in fqn for modules (e.g. pointing to a nested dir in a git repo)
```

Comment:
Here is a todo left :)

Reply:
Going to close this we still have to test how/whether generating bindings for modules works already and we're going to revisit this part of the code then.